src/main/java/dinkplugin/message/DiscordMessageHandler.java

```diff
@@ -162,6 +170,49 @@ private void sendMessage(HttpUrl url, MultipartBody.Builder requestBody, int att
         });
     }
 
+    /**
+     * Captures the next frame and applies the specified rescaling
+     * while abiding by {@link Embed#MAX_IMAGE_SIZE}.
+     *
+     * @param drawManager  {@link DrawManager}
+     * @param scalePercent {@link DinkPluginConfig#screenshotScale()} divided by 100.0
+     * @return future of the image byte array by the image format name
+     * @apiNote scalePercent should be in (0, 1]
+     * @implNote the image format is either "png" (lossless) or "jpeg" (lossy), both of which can be used in MIME type
+     */
+    private static CompletableFuture<Map.Entry<String, byte[]>> captureScreenshot(DrawManager drawManager, double scalePercent) {
+        CompletableFuture<Image> future = new CompletableFuture<>();
+        drawManager.requestNextFrameListener(future::complete);
+        return future.thenApply(ImageUtil::bufferedImageFromImage)
+            .thenApply(input -> Utils.rescale(input, scalePercent))
+            .thenApply(image -> {
+                try {
+                    String format = "png"; // lossless
```

Comment:
Should there be a config setting to always use lossy over lossless (would typically imply no rescaling necessary)